docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+services:
+  database:
+    container_name: dhis2-database
+    # image: mdillon/postgis:9.6
+    image: ethiopia-sdg:1.0
+    command: postgres -c max_locks_per_transaction=100
+    # volumes: 
+    # - ./config/init.sql:/docker-entrypoint-initdb.d/init.sql
+    environment:
+      POSTGRES_USER: dhis
+      POSTGRES_DB: dhis
+      POSTGRES_PASSWORD: dhis
```

Comment:
This sensitive information should go to the .env file with further instructions updated on the README file